applications/nrf_desktop/sysbuild/CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ set(ipc_radio_APPLICATION_CONFIG_DIR
     CACHE INTERNAL "Application configuration dir controlled by sysbuild"
 )
 
+if (BOARD MATCHES "nrf52810dmouse/nrf52810" OR
+    BOARD MATCHES "nrf52820dongle/nrf52820" OR
+    BOARD MATCHES "nrf52833dk/nrf52820")
+  set(SNIPPET bt-ll-sw-split ${SNIPPET} CACHE STRING "" FORCE)
```

Comment:
note: where is this snippet located?

Reply:
The snippet was added to `sdk-zephyr`: https://github.com/nrfconnect/sdk-zephyr/pull/1940/commits/b3ae3a624ea44582f2798e243a0d5465bceda78a